buildspec.yaml

```
@@ -10,6 +10,14 @@ phases:
       # EnvironmentVariable config for the Pipeline action :(
       - export AWS_VERSION=$(aws --version)
 
+      # Install dependencies for testing
```

I was thinking that it would be good to run the tests in the `pre-build` step so that it can fail early, so I added the `npm run test` command here.